storage/tests/unit/test_client.py

```diff
@@ -821,3 +896,39 @@ def dummy_response():
         self.assertEqual(page.remaining, 0)
         self.assertIsInstance(bucket, Bucket)
         self.assertEqual(bucket.name, blob_name)
+
+
+class _Connection(object):
```

Comment:
Analogy to class from bucket tests

Reply:
I'd prefer that we use mocks rather than add this class.

See:

https://github.com/googleapis/google-cloud-python/blob/0e231cab5f066ae5b3e254d0cb7d204e3dac7ebc/bigquery/tests/unit/helpers.py#L16-L24

and how it is used:

https://github.com/googleapis/google-cloud-python/blob/0e231cab5f066ae5b3e254d0cb7d204e3dac7ebc/bigquery/tests/unit/test_client.py#L169-L173

Reply:
Redone with similar mock